server/command/install.go

```diff
@@ -61,7 +90,7 @@ func (s *service) executeInstall(params *params) (*model.CommandResponse, error)
 	// Finish the installation when the Dialog is submitted, see
 	// <plugin>/http/dialog/install.go
 	err = s.api.Mattermost.Frontend.OpenInteractiveDialog(
-		dialog.NewInstallAppDialog(manifest, appSecret, conf.PluginURL, params.commandArgs))
+		dialog.NewInstallAppDialog(app.Manifest, app.Secret, conf.PluginURL, params.commandArgs))
```

Comment:
app.Manifest or manifest? Is app.Manifest defined in this context?

Reply:
Same consideration on line 105.

Reply:
Yes, `ProvisionApp` returns the app with all the context.